test/types/type_variables/ferguson/some-defaults-ignored.good

```diff
@@ -1,5 +1,6 @@
 some-defaults-ignored.chpl:19: warning: unstable type construction with some generic defaults
 some-defaults-ignored.chpl:1: note: this type construction might ignore field defaults in the type declared here but that may change
+some-defaults-ignored.chpl:19: warning: please add '(?)' to type 'call_tmp' because it is generic
```

Comment:
Hmm... "type `call_tmp`" doesn't seem so good.  Maybe we need to squash the name of the time if it has the "I'm a compiler temp" flag attached to it?

Reply:
Agreed. I have already fixed this, now it says `to the type 'R'`